prompt_toolkit/shortcuts/dialogs.py

```diff
@@ -111,14 +111,14 @@ def message_dialog(title='', text='', ok_text='Ok', style=None, async_=False):
 
 
 def radiolist_dialog(title='', text='', ok_text='Ok', cancel_text='Cancel',
-                     values=None, style=None, async_=False):
+                     values=None, style=None, async_=False, default=0):
```

Comment:
I like the ``default`` argument, but I would prefer if it was not an integer, but a key from the ``values`` list. Would it be possible to change it that way. The default can be ``None`` which always takes the first value.

Reply:
I'd hesitate because the input is a list of tuples. There is no guarantee of uniqueness on the values or key. Maybe also accept the tuple from the list? And do a check internally and decide what to do?

```python
if default is None:
  self.current_value = 0
elif isinstance(default, tuple):
  self.current_value = values.index(default)
else:
  self.current_value = default
```